test/extended/util/framework.go

```diff
@@ -40,13 +40,79 @@ var TestContext e2e.TestContextType
 
 const pvPrefix = "pv-"
 
+//FixOpenShiftNamespaceImageStreams is a temporary workaround for the intermittent
+// issue seen in extended tests where *something* is deleteing the pre-loaded, languange
+// imagestreams from the OpenShift namespace
+func FixOpenShiftNamespaceImageStreams(oc *CLI) {
```

Comment:
@liggitt and I agreed offline to convert the recreate imagestreams into the fail the test immediately idea I mentioned previously.  Also got a clue that the on/off switch for audit logging is in the master cfg.